tests/simulator/test_advance.py

```diff
@@ -356,7 +358,8 @@ def _test_overlapped_particledatas_have_identical_particles(
 
             overlaps = hierarchy_overlaps(datahier, coarsest_time)
 
-            for ilvl, lvl in datahier.patch_levels.items():
+            # should it be levels(coarsest_time)?
```

Comment:
no, but the dict val is not used. it just loops over all levels numbers, which is constant due to refinement boxes